README.rst

```diff
@@ -2,8 +2,17 @@
                             Tarantool C client libraries
 -------------------------------------------------------------------------------
 
-.. image:: https://travis-ci.org/tarantool/tarantool-c.svg?branch=master
-    :target: https://travis-ci.org/tarantool/tarantool-c
+.. image:: https://img.shields.io/github/workflow/status/tarantool/tarantool-c/fast_testing/master?label=Linux%20test
+    :target: https://github.com/tarantool/tarantool-c/actions/workflows/fast_testing.yml?query=branch%3Amaster
+    :alt: GitHub Workflow Status (branch)
+
+.. image:: https://img.shields.io/github/workflow/status/tarantool/tarantool-c/osx_testing/master?label=OS%20X%20test
+    :target: https://github.com/tarantool/tarantool-c/actions/workflows/osx_testing.yml?query=branch%3Amaster
+    :alt: GitHub Workflow Status (branch)
+
+.. image:: https://img.shields.io/github/workflow/status/tarantool/tarantool-c/packaging/master?label=build
```

Comment:
I would replace `?label=build` with `?label=Packaging`.

Reply:
It seems reasonable. Fixed.